### PR TITLE
UI :- Added Data Insights in elasticsearch for re-indexing

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue :- #9289
- Added Data Insights in elasticsearch for re-indexing

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
 @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/constants/elasticsearch.constant.ts [hunk 1]
```diff
@@ -48,6 +48,18 @@ export const ELASTIC_SEARCH_INDEX_ENTITIES = [
...
+    label: 'Entity Data',
+  },
+  {
+    value: 'webAnalyticEntityViewReport',
+    label: 'Web Analytic Entity',
+  },
+  {
+    value: 'webAnalyticUserActivityReport',
+    label: 'Web Analytic User',
+  },
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/constants/elasticsearch.constant.ts:62]: Please confirm name with @ShilpaVernekar once!

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexModal.component.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ const ReIndexAllModal = ({
             onChange={(values) => setEntities(values as string[])}>
             <Row gutter={[16, 16]}>
               {ELASTIC_SEARCH_INDEX_ENTITIES.map((option) => (
-                <Col key={option.value} span={6}>
+                <Col key={option.value} span={8}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexModal.component.tsx:75]: @Ashish8689 can you also make re-create index initial value to `true`?

**Comment by Ashish8689** [comment]
@ShaileshParmar11 @chirag-madlani Changes updated!

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexModal.component.tsx [hunk 3]
```diff
@@ -57,10 +57,7 @@ const ReIndexAllModal = ({
         layout="vertical"
         name="elastic-search-re-index"
         onFinish={onSave}>
-        <Form.Item
-          initialValue={false}
-          label="Recreate indexes"
-          name="recreateIndex">
+        <Form.Item initialValue label="Recreate indexes" name="recreateIndex">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/ElasticSearchReIndexModal.component.tsx:60]: localisation please

File: openmetadata-ui/src/main/resources/ui/src/constants/elasticsearch.constant.ts [hunk 4]
```diff
@@ -48,6 +48,18 @@ export const ELASTIC_SEARCH_INDEX_ENTITIES = [
     value: 'tag',
     label: 'Tag',
   },
+  {
+    value: 'entityReportData',
+    label: 'Data Assets Report',
+  },
+  {
+    value: 'webAnalyticEntityViewReport',
+    label: 'Web Analytics Report',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/constants/elasticsearch.constant.ts:57]: localisation

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

**Comment by Ashish8689** [comment]
@chirag-madlani added localisation in the files.

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Comment by chirag-madlani** [comment]
DO NOT MERGE THIS UNTIL BACKEND CHANGES PUSHED 
cc: @mohityadav766

**Review by chirag-madlani — APPROVED** [review]

**Comment by mohityadav766** [comment]
Backend side changes are done.  Cc @chirag-madlani

**Pull request merged**